tests/modules/cooler/cload/test.yml

```diff
@@ -1,29 +1,23 @@
 - name: cooler cload test_cooler_cload_pairix
-  command: nextflow run ./tests/modules/cooler/cload -entry test_cooler_cload_pairix -c ./tests/config/nextflow.config -c ./tests/modules/cooler/cload/nextflow.config
+  command: nextflow run ./tests/modules/cooler/cload -entry test_cooler_cload_pairix -c ./tests/config/nextflow.config  -c ./tests/modules/cooler/cload/nextflow.config
   tags:
     - cooler/cload
     - cooler
   files:
     - path: output/cooler/test_pairix.2000000.cool
-    - path: output/cooler/test_pairix.bedpe
-      md5sum: 0cd85311089669688ec17468eae02111
 
 - name: cooler cload test_cooler_cload_pairs
-  command: nextflow run ./tests/modules/cooler/cload -entry test_cooler_cload_pairs -c ./tests/config/nextflow.config -c ./tests/modules/cooler/cload/nextflow.config
+  command: nextflow run ./tests/modules/cooler/cload -entry test_cooler_cload_pairs -c ./tests/config/nextflow.config  -c ./tests/modules/cooler/cload/nextflow.config
   tags:
     - cooler/cload
     - cooler
   files:
     - path: output/cooler/test_pairs.2000000.cool
-    - path: output/cooler/test_pairs.bedpe
```

Comment:
Should we keep the check for the presence of the file?

Reply:
I reinstated the check on the .bedpe file, since it's the one that remains identical between runs